cocotb/share/makefiles/Makefile.pylib.Msys

```diff
@@ -47,3 +47,11 @@ ifeq ($(PYTHON_LIBDIR),None)
 endif
 
 PYTHON_INCLUDEDIR := $(shell $(PYTHON_BIN) -c 'import distutils.sysconfig; print( distutils.sysconfig.get_python_inc() )')
+
+# For Tcl script purposes, generate source file paths in the form of C:/User/...
+ifneq ($(VERILOG_SOURCES),)
+    VERILOG_SOURCES := $(shell cygpath -m $(VERILOG_SOURCES))
```

Comment:
I worry this will handle spaces poorly. Maybe that's already a lost cause.

Reply:
If there are spaces in the path it does not work in any case (with or without this change).

Reply:
Possibly for different reasons.

Reply:
I think this is fine to ignore